models/operator_property.go

```diff
@@ -0,0 +1,115 @@
+// Code generated by go-swagger; DO NOT EDIT.
+
+package models
+
+// This file was generated by the swagger tool.
+// Editing this file might prove futile when you re-run the swagger generate command
+
+import (
+	"encoding/json"
+
+	"github.com/go-openapi/errors"
+	"github.com/go-openapi/strfmt"
+	"github.com/go-openapi/swag"
+	"github.com/go-openapi/validate"
+)
+
+// OperatorProperty operator property
+//
+// swagger:model operator-property
+type OperatorProperty struct {
+
+	// Default value for the property
+	DefaultValue string `json:"default_value,omitempty"`
+
+	// Description of a property
+	Description string `json:"description,omitempty"`
+
+	// Name of the property
+	Name string `json:"name,omitempty"`
+
+	// Values to select from
+	Options []string `json:"options"`
+
+	// Type of the property
+	// Enum: [boolean string number]
+	Type string `json:"type,omitempty"`
+}
+
+// Validate validates this operator property
+func (m *OperatorProperty) Validate(formats strfmt.Registry) error {
+	var res []error
+
+	if err := m.validateType(formats); err != nil {
+		res = append(res, err)
+	}
+
+	if len(res) > 0 {
+		return errors.CompositeValidationError(res...)
+	}
+	return nil
+}
+
+var operatorPropertyTypeTypePropEnum []interface{}
```

Comment:
s/operatorPropertyTypeTypePropEnum/operatorPropertyTypePropEnum perhaps?

Reply:
That's generated, so not much influence over it.

Reply:
yeah, I figured it. But still the fact that it repeats `Type` is odd, to say the least. Just wondered if there was a way to rename it in the specs in such a way to avoid this repetition.

Reply:
Managed to have `operatorPropertyTypeDataTypePropEnum` when the property is called `data_type`